components/card.css

```diff
@@ -0,0 +1,114 @@
+.p-1 {
+  padding: 1rem;
+}
+.pt-1 {
+  padding-top: 1rem;
+}
+.py-1 {
+  padding: 1rem 0px;
+}
+
+.mb-2 {
+  margin-bottom: 2rem;
+}
+
+.card {
+  width: 50%;
```

Comment:
instead of using fixed width in % you can use min-width and max-width along with rems

Reply:
Thank you create suggested change.